.buildkite/pipeline.yml

```diff
@@ -80,11 +80,13 @@ steps:
       - cp -r testdata/* ${FC_TEST_DATA_PATH}
       # Install tc-redirect-tap.
       - 'go get github.com/awslabs/tc-redirect-tap/cmd/tc-redirect-tap'
-      # Copy vmlinux and root-drive.img.
-      - ln -s /var/lib/fc-ci/vmlinux.bin ${FC_TEST_DATA_PATH}/vmlinux
-      - ln -s /var/lib/fc-ci/rootfs.ext4 ${FC_TEST_DATA_PATH}/root-drive.img
       # Download Firecracker and its jailer.
       - make deps
+      # Copy vmlinux and root-drive.img.
+      - mkdir -p /var/lib/fc-ci
+      - 'sudo cp ${FC_TEST_DATA_PATH}/vmlinux /var/lib/fc-ci/vmlinux.bin'
+      - 'sudo cp ${FC_TEST_DATA_PATH}/root-drive.img /var/lib/fc-ci/rootfs.ext4'
```

Comment:
This is not a good idea. The host will have multiple test runs in parallel. Changing /var/lib/fc-ci from the multiple tests will create a race condition.

Reply:
The reason why I use "cp" instead of creating symbolic link is that symbolic link doesn't use the update-to-date vmlinux kernel. I am not sure why it happened.

Reply:
From [this build](https://buildkite.com/firecracker-microvm/firecracker-go-sdk/builds/1719#ea7eb7a6-2ae4-4677-8a2f-6c655046aec9), "FaultMessage:Device is inactive, check balloon driver is enabled." Which means the lates vmlinux kernel is not used.